_posts/2019-01-16-lecture-02.md

```diff
@@ -0,0 +1,459 @@
+
+#### Types of active two edge trails 
+
+By examining 2-edge connections between nodes $$X,Y,Z$$ four types of active trails, i.e. trails along which influence can flow from $$X$$ to $$Z$$, are apparent:
+
+1. Causal trail $$X \rightarrow Z \rightarrow Y$$ : active iff Z is not observed. 
+2. Evidential trail $$X \leftarrow Z \leftarrow Y$$ : active iff Z is not observed.
+3. Common cause $$X \leftarrow Z \rightarrow Y$$ : active iff Z is not observed.
+4. Common effect $$X \rightarrow Z \leftarrow Y$$ : active iff Z or one of its descendants is observed.
+
+For influence to flow through a longer trail $$ X_1 \rightleftharpoons ... \rightleftharpoons X_n  $$, every two-edge trail within must allow influence flow. This is summarized as follows:
+
+#### Definition 3.6 active trail
+Let $$\mathcal{G}$$ be a BN structure and. $$ X_1 \rightleftharpoons  ... \rightleftharpoons X_n  $$ be a trail in $$\mathcal{G}$$. Let $$Z$$ be a subset of observed variables. Then the trail $$ X_1 \rightleftharpoons  ... \rightleftharpoons  X_n$$ is active given $$Z  $$ if:
+
+* Whenever we have a v-structure $$ X_{I-1}  \to X_i \leftarrow X_{I+1}  $$ then $$X_i$$ or one of its descendants are in $$Z$$
+* No other node along the trail is in $$Z$$
+
+Further, for directed graphs that may have more than one trail between nodes "directed separation" (d-separation) gives a notion of separation between nodes.
+
+#### Definition 3.7 D-separation 
+Let $$X,Y,Z$$ be three sets of nodes in $$\mathcal{G}$$. We say that *$$X, Y$$* are d-separated given $$Z$$ (denoted $$ \text{d-sep}_{\mathcal{G}} (X; Y \vert Z) $$ ) if there is no active trail between any node X $$ \in X $$ and Y $$ \in Y $$ given $$Z$$. 
+
+### 3.3.2 Soundness and Completeness
+As a method d-separation has the following properties (proof in text):
+
+* _Soundness_: $$ X, Y $$ are d-separated given $$ Z \implies X \perp Y \vert Z $$ i.e. independence determined by d-separation is satisfied by the underlying distribution. 
+* _Completeness_:  If $$ X \perp Y \vert Z $$ then they are d-separated, i.e. d-separation detects all possible independencies.
+
+***
+
+# In class notes
+
+Motivation: representing joint distributions over some random variables is computationally expensive, so we need methodologies to represent joint distributions compactly.
+
+## Two types of Graphical Models <d-cite key="Blei:2015:BasicsPGM"></d-cite><d-cite key="Jordan:2003:PGM"></d-cite>
+### Directed Graphs (Bayesian Networks)
+An acyclic graph, $$\mathcal{G}$$, is made up of a set of nodes, $$\mathcal{V}$$, and a set 
+of directed edges, $$\mathcal{E}$$, where edges represent a causality relationship between nodes. Nodes in the graph represent a set of random variables, $$\{x_1,...,x_N\}$$, where there is a one-to-one map between nodes and random variables. 
+
+<img src="{{ '/assets/img/notes/lecture-02/directed-graph.png' | relative_url }}" height ="270" width = "480"/>
```

Comment:
Please use `<figure>...</figure>` tags as given in [the template](https://sailinglab.github.io/pgm-spring-2019/notes/lecture-notes-template/#figures). (For all your figures below as well.)